Makefile

```diff
@@ -274,7 +274,7 @@ remotesystem:
 	if timeout -v 1 true; then \
 		SOCK_FILE=$(shell mktemp --dry-run --tmpdir io.podman.XXXXXX);\
 		export PODMAN_VARLINK_ADDRESS=unix:$$SOCK_FILE; \
-		./bin/podman varlink --timeout=0 $$PODMAN_VARLINK_ADDRESS &>/dev/null & \
+		./bin/podman varlink --timeout=0 $$PODMAN_VARLINK_ADDRESS |& tail -100 >/tmp/varlink.log & \
```

Comment:
Why not just &>? varlink is verbose, but not megabytes-verbose.

Reply:
oh gosh, I was thinking terabyte-verbose from your description :smile:   Yep, can probably just dump to a file.